scvi/__init__.py

```diff
@@ -3,12 +3,6 @@
 # Set default logging handler to avoid logging with logging.lastResort logger.
 import logging
 
-try:
-    # necessary as importing scvi after ray causes kernel crash
-    from ray import tune  # noqa
-except ImportError:
-    pass
-
```

Comment:
Does it work now without this block?

Reply:
I was having no issues. Can you check as well?

Reply:
Yeah I'm still having the issue unfortunately. I usually check by running
```
import scvi
import ray
ray.init()
ray.shutdown()
```

Reply:
I'm not having that issue and the tests are passing, I'm wondering what could be happening